src/screens/factories/factories.tsx

```diff
@@ -0,0 +1,124 @@
+"use client";
+
+import { Experimental_CrudDataTable } from "@/widgets";
+import { ColumnDef, PaginationState } from "@tanstack/table-core";
+import { Factory } from "@/entities/factory";
+import { FiltersState, GuidWithCopy } from "@/shared/ui";
+import { UserResourceLink } from "@/entities/user";
+import { useContext } from "react";
+import { ApiContext, PagedResponse } from "@/shared/api";
+import { toast } from "sonner";
+
+const colDefs: Record<
+  keyof Omit<Factory, "createdByUserId">,
+  ColumnDef<Factory>
+> = {
+  id: {
+    id: "id",
+    accessorKey: "id",
+    header: "ID",
+    cell: (data) => <GuidWithCopy value={data.getValue<string>()} />,
+  },
+  key: {
+    id: "key",
+    accessorKey: "key",
+    header: "Key",
+  },
+  description: {
+    id: "description",
+    accessorKey: "description",
+    header: "Description",
+  },
+  name: {
+    id: "name",
+    accessorKey: "name",
+    header: "Name",
+  },
+  createdAt: {
+    id: "createdAt",
+    accessorKey: "createdAt",
+    header: "Created At",
```

Comment:
let `import { formatToTwinfaceDate } from "@/shared/libs";` and use `formatToTwinfaceDate` to format `createdAt`